content/Certification/Guides/PICSandPIXIT.md

```diff
@@ -0,0 +1,199 @@
+tool are distributed as part of the official specification release package and
+are available to members on
+[Causeway](https://groups.csa-iot.org/wg/members-all/document/folder/2269)
+(requires CSA member login).
+
+## PIXITs
+
+PIXIT stands for Protocol Implementation eXtra Information for Testing. A PIXIT
+value provides an implementation-defined condition or value that is required by
+the test.
+
+PIXIT values are used to convey information that is required for testing, but
+not normally available to a client interacting with the device. For example, the
+key for a test event trigger on the device, or the network credentials for the
+test harness.
+
+More information on PIXITs and the PIXIT format can be found in the
+[PICS Guidelines](https://github.com/CHIP-Specifications/chip-test-plans/blob/master/docs/PICS%20Guidelines.md).
+
+## Creating PICS conformance statements for Matter devices
+
+Because Matter devices may contain multiple, differing instances of the same
+cluster on multiple endpoints, it it not possible to unambiguously describe a
+Matter device with a single set of cluster-based PICS files. Instead, Matter
+devices use a full set of PICS XML files to describe each endpoint.
+
+To create a conformance statement for a Matter device, for each endpoint, load
+the full set of PICS XML files into the
+[PICS tool](https://picstool.csa-iot.org/), and select each of the PICS elements
+present for the endpoint being described. Documentation on how to use the PICS
+tool can be found in the PICS tool readme in the tool menu.
+
+Some of the full-node MCORE elements really only apply to the root node,
+while others apply across all endpoints. For example, the entire device is
+commissionable, but commissioning tests only need to be run against EP0, so
+MCORE PICS should be set on the EP0 PICS set. Things like MCORE.IDM apply to
+every endpoint.
+
+
+### Helper scripts
+
+The official tooling for CSA certification is the PICS tool provided as a part
+of the release. PICS files need to pass validation on the PICS tool to be valid
+for certification.
+
+However, in Matter, many of the PICS codes correspond directly to elements that
+are exposed directly on the device. For example, cluster presence is determined
+from the server list on an endpoint. Feature maps and attribute and command lists
+correspond directly to the PICS.S.Fxx, PICS.S.Axxxx and PICS.S.Cxx.Xx PICS
+codes. The Matter SDK provides a tool to pre-fill these values in the PICS XML
+files so they do not have to be individually, manually filled in the PICS tool.
+
+-   [PICS Generator](https://github.com/project-chip/connectedhomeip/tree/master/src/tools/PICS-generator)
+
+PICS codes filled using this tool still need to be validated by the PICS tool.
+Note that due to device limitations, the tool will NOT fill the following
+categories of PICS codes, and these will need to be filled manually in the PICS
+tool:
+
+-   Event PICS (PICS.S.Exx)
```

Comment:
It is actually only optional or conditional events. Mandatory events is set by the tool.

```suggestion
-   Optional or conditional event PICS (PICS.S.Exx)
```